build/yamls/patches/netattachdef/net-attach-def.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I think this file needs to be removed? Comments have been addressed and it doesn't seem to be used anywhere anymore.